README.md

```diff
@@ -199,6 +212,15 @@ Example
     python analyze.py --train_file_path=~/tacotron/training/train.txt --save_to=~tacotron/visuals --cmu_dict_path=~/cmudict-0.7b
 ```
 
+<<<<<<< HEAD
```

Comment:
Looks like some artifacts got left behind from a merge conflict. If you delete these we should be good to go :)

Thanks heaps for all the work on this!

Reply:
Hopefully all unneeded artifacts should be removed now.